lib/run/index.js

```diff
@@ -133,10 +133,33 @@ module.exports = function (options, callback) {
                 result[value] = true;
             }, {});
         }
+        let RequiredFolder = [];
 
+        function checkItems (item) {
+            item.members.forEach((Element) => {
+                if (Element.items) {
+                    RequiredFolder.push(Element.name);
+                    checkItems(Element.items);
+                }
+                else {
+                    RequiredFolder.push(Element.name);
+                }
+            });
+        }
         // sets entrypoint to execute if options.folder is specified.
         if (options.folder) {
             entrypoint = { execute: options.folder };
+            checkItems(options.collection.items);
+            if (_.isArray(options.folder)) {
+                options.folder.map((e) => {
+                    if (!RequiredFolder.includes(e)) {
+                        console.warn('Folder ', e, ' is not found');
+                    }
+                });
+            }
+            else if (!RequiredFolder.includes(options.folder)) {
+                console.warn('Folder ', options.folder, ' is not found');
```

Comment:
Since we are adding on to the errors now for general Newman run (not CLI reporter), the visual formatting / UX of the same comes into question.